main_test.go

```diff
@@ -36,6 +36,7 @@ var mainTests = []mainTest{
 	{"simple_arith1", "3 + 3", "6", ""},
 	{"simple_arith2", "3 + 3 + 3", "9", ""},
 	{"simple_arith3", "(3 + 3) + (3 + 3)", "12", ""},
+	{"simple_arith_string", "\"aaa\" + \"bbb\"", "\"aaabbb\"", ""},
```

Comment:
However, this is not the primary place for tests. We'll probably share the test suite with the C++ version with examples in separate files. We don't do that yet, because command line interface is not implemented yet.